components/ui/Tag.jsx

```diff
@@ -0,0 +1,11 @@
+const Tag = ({ content }) => {
+  return (
+    <div>
+      <p className="font-neuf text-lg text-primaryGray-400  -tracking-wide  bg-primaryGray-200  rounded-base  px-2.5  py-sm w-fit">
```

Comment:
The padding should be adjusted according to the design like so, `padding: 3px 10px;`